product.json

```diff
@@ -35,6 +35,55 @@
 	"win32ShellNameShort": "C&ode - OSS",
 	"darwinBundleIdentifier": "com.visualstudio.code.oss",
 	"linuxIconName": "com.visualstudio.code.oss",
+	"extensionsGallery": {
+		"nlsBaseUrl": "https://www.vscode-unpkg.net/_lp/",
```

Comment:
Mistakenly added to PR?

Reply:
Hm, looks like the husky pre commit hooks did not run in the codespace I authored this PR in even though husky is in our dev dependencies. Thanks!